assets/cms/components/form/ConcreteFileDirectoryInput.vue

```diff
@@ -122,10 +122,10 @@ export default {
             searchField: 'directoryName',
             render: {
                 option: function (data, escape) {
-                    return `<div class="level-${data.directoryLevel}"><i class="fa fa-folder"></i> ${data.directoryName}</div>`
+                    return `<div class="level-${data.directoryLevel}"><i class="fa fa-folder"></i> ${data.directoryName.replace(/[^a-zA-Z0-9-_ ]/g, '')}</div>`
```

Comment:
What about multimeter chars (eg Japanese or accented letters)? What if we have "A & B" or "A > B"

Reply:
Alrighty what about this https://github.com/concretecms/bedrock/pull/370/commits/4a0f858dc039b14a6da09cc0b953f463fffeab3f ?

Reply:
That's perfect!